handlers/main.yml

```diff
@@ -2,4 +2,9 @@
 - name: reload pm2-{{ enketo_user }}
   service:
     name: "pm2-{{ enketo_user }}"
-    state: reloaded
\ No newline at end of file
+    state: reloaded
+
+- name: reload nginx
+  debug:
+    msg: "dummy reload nginx task"
+  when: test
```

Comment:
Do we need this in the role?

Reply:
it seems we make calls to this handler yet it does not exist in this role. for example here https://github.com/onaio/ansible-enketo/blob/4f5fb8a19868092a5b1f2dedcb512a605580f611/tasks/install-enketo.yml#L73. This caused molecule tests to fail. so a dummy handler like this works when `test` is set to `true`